cluster/senza-definition.yaml

```diff
@@ -478,6 +478,17 @@ Resources:
         - Key: "kubernetes.io/cluster/{{ Arguments.ClusterID }}"
           Value: owned
     Type: AWS::EC2::SecurityGroupIngress
+  WorkerSecurityGroupIngressFromWorkerTocAdvisor:
```

Comment:
I think you could also add this to [this list](https://github.com/zalando-incubator/kubernetes-on-aws/blob/dev/cluster/senza-definition.yaml#L414).